pessimistictest/pessimistic_test.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2019 PingCAP, Inc.
```

Comment:
Can this test be moved to ```tests `` folders?

Reply:
Is there any specific reason we would like to do the move? The current tests in `tests` folds are those of 'integration tests'.